test.tz

```diff
@@ -0,0 +1,25 @@
+{ parameter
+    (big_map
```

Comment:
No, there should be 1 big_map per entrypoint. So in this case there should be 2 big_maps.